<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_InsertTable</class>
 <widget class="QDialog" name="Dialog_InsertTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LaTeX Office Writer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="4" colspan="3">
    <widget class="QSpinBox" name="spinBoxRows">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="QTableWidget" name="tableWidget">
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>Row 1</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Column 1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="labelRows">
     <property name="text">
      <string>Rows</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QSpinBox" name="spinBoxColumns">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelColumns">
     <property name="text">
      <string>Columns</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelHeaderColor">
     <property name="text">
      <string>Header Color</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelReference">
     <property name="text">
      <string>Table Reference</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="6">
    <widget class="QLineEdit" name="lineEditReference"/>
   </item>
   <item row="7" column="0" colspan="7">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelCaption">
     <property name="text">
      <string>Table Caption</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="6">
    <widget class="QLineEdit" name="lineEditCaption"/>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QComboBox" name="comboBoxColor">
     <item>
      <property name="text">
       <string>White</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gray</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Red</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Blue</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Green</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Yellow</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pink</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="labelPosition">
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5" colspan="2">
    <widget class="QComboBox" name="comboBoxPosition">
     <item>
      <property name="text">
       <string>Here</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Top</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bottom</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_InsertTable</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_InsertTable</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
